src/helper.php

```diff
@@ -33,6 +36,30 @@ function tinker(...$args)
             ->combine($args)
             ->all();
 
-        exec('open "tinkerwell://?cwd='.base64_encode(base_path()).'&scope='.base64_encode(serialize($namedParameters)).'"');
+        //exec('open "tinkerwell://?cwd='.base64_encode(base_path()).'&scope='.base64_encode(serialize($namedParameters)).'"');
+
+        // For MacOS
+        $command = 'open "tinkerwell://?cwd='.base64_encode(base_path()).'&scope='.base64_encode(serialize($namedParameters)).'"';
+
+        if (strtoupper(substr(PHP_OS, 0, 3)) === 'WIN') {
```

Comment:
As this package requires `illuminate/support`, you could use:
```suggestion
        if (windows_os()) {
```